docker-compose.override.yml

```diff
@@ -37,6 +35,7 @@ services:
     volumes:
       - ./packages/app:/usr/src/app
       - /usr/src/app/node_modules
+      - /usr/src/app/.next
```

Comment:
Ensures the .next folder is mounted correctly

Reply:
Nice fix!